internal/auth/authenticator_ldap.go

```diff
@@ -124,11 +137,15 @@ func (la *LDAPAuthenticator) Authenticate(msg *spoe.Message) (bool, []spoe.Actio
 		return false, nil, fmt.Errorf("unable to parse basic auth header")
 	}
 
-	err = verifyCredentials(&la.connectionDetails, username, password)
+	err = verifyCredentials(&la.connectionDetails, username, password, group)
 
 	if err != nil {
 		if err == ErrUserDoesntExist {
-			logrus.Debugf("user %s does not exist", username)
+			if is_group_provided {
+				logrus.Debugf("user <%s> from group <%s> does not exist", username, group)
```

Comment:
A bit ambiguous, we should tell somehting like "user doesn't exist, or is not member of group x".

Reply:
Updated